robottelo/api/utils.py

```diff
@@ -406,15 +406,20 @@ def configure_provisioning(org=None, loc=None, compute=False):
         query={u'search': u'name="x86_64"'}
     )[0].read()
 
+    #
+    media = entities.Media(location=[loc], organization=[org]).create()
```

Comment:
real media should be provided automatically, not the fake one

Reply:
@oshtaier @sghai Ok, this was fixed for 6.2 but never was backported to master, I mean for 6.2 there is added media while code in master has nothing related to media. I'll backport that a little bit later.